frontend/src/components/global/Footer.js

```diff
@@ -1,21 +1,24 @@
-import React from "react";
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faInstagram } from '@fortawesome/free-brands-svg-icons';
+import { faFacebookF } from '@fortawesome/free-brands-svg-icons';
+import { faTwitter } from '@fortawesome/free-brands-svg-icons';
```

Comment:
these 3 lines can be combined as
import { faInstagram, faFacebookF, faTwitter } from '@fortawesome/free-brands-svg-icons';

Reply:
good to know! this as been condensed